prime-dotnet-webapi/Services/CommunitySiteService.cs

```diff
@@ -151,28 +156,39 @@ public async Task UpdateSiteAsync(int siteId, CommunitySiteUpdateModel updatedSi
 
             _context.Entry(currentSite).CurrentValues.SetValues(updatedSite);
 
+            var updateDetail = new List<string>();
             if (currentSite.SubmittedDate == null)
             {
-                UpdateVendors(currentSite, updatedSite);
+                var vendors = await GetVendorsAsync();
+                UpdateVendors(currentSite, updatedSite, vendors);
```

Comment:
`UpdateVendors` returns change details but they are discarded?

Reply:
Good catch. It is updated.

Reply:
The change has a side-effect 
![image](https://github.com/bcgov/moh-prime/assets/7586665/499ec53d-e527-471a-8112-c2427f59dbd6)

Reply:
@bergomi02 FYI I'm going to revert the change, add comment, and continue PR review